packages/@o3r/workspace/project.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "workspace",
+  "$schema": "https://raw.githubusercontent.com/nrwl/nx/master/packages/nx/schemas/project-schema.json",
+  "projectType": "library",
+  "sourceRoot": "packages/@o3r/workspace/src",
+  "prefix": "o3r",
+  "targets": {
+    "build": {
+      "executor": "nx:run-script",
+      "outputs": [
+        "{projectRoot}/dist/package.json"
+      ],
+      "options": {
+        "script": "postbuild"
+      },
+      "dependsOn": [
+        "compile",
+        "build-builders"
+      ]
+    },
+    "prepare-build-builders": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "prepare:build:builders"
+      }
+    },
+    "compile": {
+      "executor": "@nx/js:tsc",
```

Comment:
I think this executor is adding `type: module` in the package.json.
For ama-sdk/core we discarded it in the favour of `nx:run-script` -> script using `tsc`.

Reply:
nop, it is not here thanks to the option `"module": "CommonJS"` of the `tsconfig.build.json`